CHANGES.md

```diff
@@ -9,6 +9,7 @@ Change Log
   *
 * Fixes Cesium.js failing to parse in IE 8 and 9. While Cesium doesn't work in IE versions less than 11, but this allows for more graceful error handling.
 * Fix calling `Scene.pickPosition` after calling `Scene.drillPick`. [#2813](https://github.com/AnalyticalGraphicsInc/cesium/issues/2813)
+* Added support for ArcGIS token-based authentication in MapServer imagery provider
```

Comment:
We try to list the class names when appropriate., so I would change "MapServer imagery provider" to "`ArcGisMapServerImageryProvider`"